plugins/redirect/redirect.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function redirect(toPath) {
```

Comment:
Probably need the license block since this is inlined from the MIT package https://github.com/nsresulta/gatsby-plugin-meta-redirect/blob/master/LICENSE#L12-L13